generic/system/metadata.yaml

```diff
@@ -3,7 +3,6 @@ policyRules:
 - name: maint-tools-access
   precondition:
   - /sbin/*
-  - OPTSCAN
```

Comment:
if we remove `OPTSCAN` flag here then we wont be able to show this policy using discovery-engine since we are not downloading and checking the container image. @yasin-cs-ko-ak

Reply:
Updated the metadata